projects/documentation/content/registry-conflicts.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: dev-mode.njk
+title: 'Registry Conflicts: Spectrum Web Components'
+displayName: Registry conflicts
+slug: registry-conflicts
+---
+
+# Registry conflicts
+
+Element names are unique, so if you try to register two different implementations of the same element via [window.customElements](https://developer.mozilla.org/en-US/docs/Web/API/Window/customElements), you'll encounter an error like:
+
+```
+Uncaught DOMException: Failed to execute 'define' on 'CustomElementRegistry':
+the name "foo-bar" has already been used with this registry
+```
+
+You might hit this error when:
+
+-   Multiple components import the redefined component as a dependency and the project's dependency tree has not been [deduped](https://docs.npmjs.com/cli/v9/commands/npm-dedupe?v=true).
+-   Multiple components import out-of-date versions of the redefined component, such that there is no [semver-valid](https://docs.npmjs.com/about-semantic-versioning) version to satisfy all of them.
+-   A package publishes a pre-built JavaScript blob instead of [import](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/import)-able files, and includes web component dependency definitions in that blob instead of listing them as external.
+
+## Resolution
+
+First, verify that each `@spectrum-web-components` package in your `package.json` shares the same version. If `@spectrum-web-components/button` is on version `0.4.0`, then all other SWC packages should _also_ be on version `0.4.0`.
```

Comment:
We know that the first version that supports this is `0.30.0` should we use that rather than one we know doesn't support this?

Reply:
Good call